lib/constellation.rb

```diff
@@ -0,0 +1,39 @@
+class ConstellationHttpException < StandardError
+end
+
+require 'net/http'
+module Constellation
+  def self.create_rsvp!(rsvp_params)
+    return unless constellation_enabled?
+    uri = URI.parse("#{app}/rsvps")
+    http = Net::HTTP.new(uri.host, uri.port)
+    unless Rails.env.development?
+      http.use_ssl = true
+      http.verify_mode = OpenSSL::SSL::VERIFY_NONE
+    end
+    req = Net::HTTP::Post.new(uri.path, headers)
+    req.body = { rsvp: rsvp_params }.to_json
+    response = http.request(req)
```

Comment:
Nice use of stdlib 👌 

If you ever need just a _little_ more, this is a tiny wrapper around `Net::HTTP` https://github.com/Fingertips/nap.